### ISSUE TITLE
Use github for version control

### ISSUE THREAD

**Issue by redndahead** [mulesoft/api-workbench#96]
Currently it seems the project is using a version control somewhere else and then replacing the entire repo here. This has the unfortunate side effect that one cannot see any of the commits that happen to the project. This makes it difficult to learn from the changes that happen in the project so developers can contribute.

Any chance of changing your process to at least push the entire commit log to github?

**Comment by sichvoge** [comment]
@redndahead that will be the plan going onwards. There is a bit more behind that project that we are working on and need a bit more time to clean these things up before fully releasing a version that we feel best to people who want to contribute. I'll keep that open and give an update on that as soon as possible.

**Comment by abaskov** [comment]
@sichvoge I support this request. Having appropriate version controlling and unobfuscated main.js would help to track progress and also will make it possible for community to contribute with PR containing fixes and suggestions.

Is there any update on timing?

**Comment by sichvoge** [comment]
I agree that it improve a lot of things. There is no real update on any timelines yet as a lot of focus currently goes into finalizing the specification and parser. That does not mean we haven't started yet, we are basically on our way along the other tasks ;)

I am really sorry that I can't give you something more specific. Although, I can assure that it is our intention to get to a full open source experience sooner rather than later.

**Comment by ProLoser** [comment]
I realize there are factors I don't understand, but I'm just going to throw this out there: In my experience a lot of OSS devs refrain from posting their code or WIP online due to concerns about clarity and organization. This has always seemed silly to me, because although it always helps to have well documented code, with a community as eager to contribute as you've got here, you may very well find that people will help you get up to speed on those sorts of things sooner rather than later.

### TL;DR
Open source now, worry about documentation/organization later.

**Comment by sichvoge** [comment]
Thank you for waiting. Happy to announce that the code is available in our [code branch](https://github.com/mulesoft/api-workbench/tree/code) now. The plan is to tidy up a bit more and fix some issues with regards to that ticket [here](https://github.com/mulesoft/api-workbench/issues/170), before merging it to our branch.
